code/script.js

```diff
@@ -1,9 +1,17 @@
 // A function that adds and remove the class "active" on the section you click on.
 // We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
+// 
+
 function toggle() {
   this.classList.toggle("active")
 }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+document.getElementById("question-1").onclick = toggle
+
+document.getElementById("question-2").onclick = toggle
+
+document.getElementById("question-3").onclick = toggle
+
+document.getElementById("question-4").onclick = toggle
```

Comment:
Nice and clean! You could also (if you'd like) break out the DOM selectors to variables, like so:
```
const question1 = document.getElementById("question-1")
question1.onclick = toggle
```